src/users/model.js

```diff
@@ -7,11 +7,7 @@ const User = connection.define("User", {
         allowNull: false,
         unique: true,
     },
-    email: {
-        type: DataTypes.STRING,
-        allowNull: false,
-        unique: true,
-    },
+
```

Comment:
we made the decision to leave email in - if your local build is not working with emails drop your tables and repopulate them